pyproject.toml

```diff
@@ -16,6 +16,7 @@ keywords = [
     "ypy",
 ]
 dependencies = [
+    "importlib_metadata >=3.6; python_version<\"3.10\"",
```

Comment:
I'm wondering if installing a package depending on a Python version will be possible in conda-forge?
Otherwise, should we always install `importlib_metadata`?
Is it better than installing `setuptools`?

Reply:
`pkg_resources` is kind of deprecated now so ideally it will be better to not re-add `setuptools`.

For reference `jupyterlab` does something similar and defines the dependency on `importlib_metadata` here: https://github.com/jupyterlab/jupyterlab/blob/8e08e4252a79a6c816535b9e80759adff984cad7/pyproject.toml#L38

But with a plain dependency in the conda recipe: https://github.com/conda-forge/jupyterlab-feedstock/blob/da1de93fbebae6ce55d98543576c04230a02d553/recipe/meta.yaml#L30

Reply:
Thanks for clarifying!